pkgs/by-name/in/inngest/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "inngest";
+  version = "1.11.2";
+
+  src = fetchFromGitHub {
+    owner = "inngest";
+    repo = "inngest";
+    rev = "v${version}";
+    hash = "sha256-5biiAozUwQlZn5VPHvon1aEOL1pc9MGFfhyCqUtvQFw=";
+  };
+
+  vendorHash = null;
+
+  subPackages = [ "cmd" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/inngest/inngest/pkg/inngest/version.Version=${version}"
+  ];
+
+  # Match goreleaser env: CGO_ENABLED=0
+  env.CGO_ENABLED = 0;
+
+  # Rename binary to match goreleaser config: binary: inngest
+  postInstall = ''
+    mv $out/bin/cmd $out/bin/inngest
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "The developer platform for easily building reliable workflows";
+    homepage = "https://www.inngest.com/";
+    license = licenses.sspl;
+    maintainers = with maintainers; [ ];
+    mainProgram = "inngest";
+  };
```

```suggestion
  meta = {
    description = "The developer platform for easily building reliable workflows";
    homepage = "https://www.inngest.com";
    license = lib.licenses.sspl;
    maintainers = with lib.maintainers; [ ];
    mainProgram = "inngest";
  };
```
`with lib` was abused in the past and is now frowned upon. Only single units (like lists) can be `with`ed